src/middlewared/middlewared/api/v25_04_0/acme_dns_authenticator.py

```diff
@@ -54,6 +54,16 @@ class CloudFlareSchemaArgs(CloudFlareSchema):
     pass
 
 
+class DigitalOceanSchema(BaseModel):
+    authenticator: Literal['digitalocean']
+    api_token: NonEmptyString = Field(description='API Token')
```

Comment:
Let's please use `Secret` as we don't want this to be dumped somewhere unsafe

Reply:
@sonicaj I added `Secret` again, changed some names for better UI and retested the current state again.
I successfully was able pass the ACME challenge and retrieve a certificate :tada: